types/transfer/transfer.d.ts

```diff
@@ -213,6 +215,26 @@ declare module 'starkbank' {
         function _delete(id: string, user?: Project | Organization): Promise<Transfer>;
         export { _delete as delete }
         
+        export class Rule {
+            /**
+             *
+             * Transfer.Rule object
+             *
+             * @description The Transfer.Rule object modify the behavior of Transfer objects created in your Workspace.
```

Comment:
Mesmo comentário lá de cima que é bom deixar a mesma docstring para o Transfer.Rule do Python. 
`The Transfer.Rule object modifies the behavior of Transfer objects when passed as an argument upon their creation.`